www/themes/Gamma/templates/frontend/profile.tpl

```diff
@@ -54,7 +58,7 @@
 		<th>Grabs Total:</th>
 		<td>{$user.grabs}</td>
 	</tr>
-	{if ($user.id==$userdata.id || $userdata.role==2) && $site->registerstatus==1}
+	{if (!$publicview || $isadmin) && $site->registerstatus==1}
```

Comment:
ok. I'll merge it as is, but I'm still unsure about it. May be alpha is wrong too ;-)